project4/3700crawler

```diff
@@ -91,14 +102,23 @@ class WebCrawler:
 
         # Send the request
         sock = self.connect(host)
-        sock.send(str.encode(make_http('GET', path, {'cookie': self.cookie, 'host':host})))
+        headers = {
+            'cookie': self.cookie,
```

Comment:
I've been capitalizing the header keys to match the formatting of the responses we've been getting (e.g., `User-Agent` instead of `user-agent`).

Reply:
Okay I'll do that. Wasn't sure if it was standardized